pudb/var_view.py

```diff
@@ -417,15 +417,15 @@ def walk_value(self, parent, label, value, id_path=None, attr_prefix=None):
             if key_it is None:
                 try:
                     len_value = len(value)
-                except TypeError:
+                except (AttributeError, TypeError):
```

Comment:
Under what circumstances does `AttributeError` happen here?
```py
>>> class A:
...    pass
... 
>>> a = A()
>>> len(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'A' has no len()
```

Reply:
You know, I'm not sure. I added that at some point during a marathon debugging session at work but forgot to copy down the traceback, if there even was one. I don't mind removing it.

Reply:
(it's possible it was even some sort of internal bug in a `__len__` implementation)

Reply:
Thanks for answering! I don't think it hurts.